File: core/src/main/java/info/nightscout/androidaps/interfaces/DanaRInterface.java
package info.nightscout.androidaps.interfaces;

import info.nightscout.androidaps.data.PumpEnactResult;

/**
 * Created by mike on 12.06.2017.
 */

public interface DanaRInterface {
    PumpEnactResult loadHistory(byte type); // for history browser
    PumpEnactResult loadEvents(); // events history to build treatments from
    PumpEnactResult setUserOptions(); // like AnyDana does
}

File: core/src/main/java/info/nightscout/androidaps/interfaces/DanaRInterface.kt
package info.nightscout.androidaps.interfaces

import info.nightscout.androidaps.data.PumpEnactResult

interface DanaRInterface {

    fun loadHistory(type: Byte): PumpEnactResult    // for history browser
    fun loadEvents(): PumpEnactResult               // events history to build treatments from
    fun setUserOptions(): PumpEnactResult           // like AnyDana does
}